### PR TITLE
Bugfix: Lombok false positive on S1258

### PR DESCRIPTION
**Pull request by cardil**
This PR fixes an false positive issue raised by analysis if one is using [Lombok Constructor annnotations](https://projectlombok.org/features/Constructor.html). The rule S1258 - `org.sonar.java.checks.AtLeastOneConstructorCheck` was not filtered properly by `org.sonar.java.filters.LombokFilter`.

Please ensure your pull request adheres to the following guidelines: 

- [x] Use the following formatting style: [SonarSource/sonar-developer-toolset](https://github.com/SonarSource/sonar-developer-toolset#code-style)
- [x] Unit tests are passing and you provided a unit test for your fix
- [x] ITs should pass : To run ITs locally, checkout the [README](https://github.com/SonarSource/sonar-java/blob/master/README.md) of the project.
- [x] If there is a [Jira](http://jira.sonarsource.com/browse/SONARJAVA) ticket available, please make your commits and pull request start with the ticket number (SONARJAVA-XXXX)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ To run integration tests, you will need to create a properties file like the one
     sonar.runtimeVersion=5.6
 
     orchestrator.updateCenterUrl=http://update.sonarsource.org/update-center-dev.properties
+    maven.localRepository=/home/bmurray/.m2/repository
```
Comment (cardil) [inline README.md]: This one is required to build pass. Sonar Orchestrator returns null, if `maven.localRepository` in not set. BTW. Orchestrator should return this path as default as auto discovered.

**Comment by m-g-sonar** [comment]
Hello @cardil , 
Thanks a lot for your contribution. I'm going to open a new PR with your commits in order to trigger our own QA. If everything goes fine, I'll merge it with our Master.

Regarding your fix, everything looks good to me. Nice work.
Cheers,

Michael

**Comment by m-g-sonar** [comment]
And merged! Note that I'll probably slightly update your change of the README.md file to make it a bit more explicit.

**Comment by cardil** [comment]
Nice. Cheers!

**Pull request merged**